spacy/tests/parser/test_parse.py

```diff
@@ -355,6 +412,7 @@ def test_overfitting_IO(pipe_name):
         train_examples.append(Example.from_dict(nlp.make_doc(text), annotations))
         for dep in annotations.get("deps", []):
             parser.add_label(dep)
+    # train_examples = train_examples[:1]
```

Comment:
Seems like a statement from a debugging session; can probably be removed.

Reply:
Removed.